docker/addons/vault/vault-env-copy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
```

Comment:
Consider renaming all `copy` files (maybe to `vault_copy_certs` and `vault_copy_env` - also, use underscore for the sake of consistency) because the `copy` suffix can be confused with accidentally copied files.

Reply:
file name replace - with underscore